docs/blog/segments-feature/index.qmd

```diff
@@ -1,9 +1,9 @@
 ---
 jupyter: python3
 html-table-processing: none
-title: "Beyond Simple Validations: Segmenting Your Data with Pointblank"
+title: "Pointblank Workshops"
```

Comment:
From pairing, we brainstormed something like, "Let's do a Pointblank workshop"

But def follow your heart